frontend-pwa/src/app/App.tsx

```
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+import { getUsers } from '../api/client';
+
+export function App() {
+  const { data } = useQuery({ queryKey: ['users'], queryFn: () => getUsers() });
+  return (
+    <div className="p-6 min-h-screen bg-base-200 text-base-content">
+      <div className="navbar bg-base-100 rounded-box mb-6">
+        <a className="btn btn-ghost text-xl">myapp</a>
```

Biome (2.1.2)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)